var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
I agree that neither solution is particularly elegant. They're both hacks. The long term plan is to have some kind of a `MetaPackage` or `BundlePackage` base class that allows you to list dependencies without having to worry about fetching or installing. I'm not sure what the progress is on that. @citibeth might know more.